### PR TITLE
fix(core-select/select-input): placeholder

### PR DESCRIPTION
**Pull request by LucasPerrault**
## Description

fix select-input placeholder of core-select

-----

### CODE REVIEW

File: packages/ng/core-select/input/select-input.component.ts [hunk 1]
```diff
@@ -104,7 +104,7 @@ export abstract class ALuSelectInputComponent<TOption, TValue> implements OnDest
 	}
 
 	public get inputPlaceholder(): string | null {
-		return this.value ? null : this.placeholder;
+		return this.value ? null : this.placeholder$.value;
```
Comment (Supamiu) [inline packages/ng/core-select/input/select-input.component.ts:107]: Not a big fan of this, we should use `| async` in the template instead.

**Review by Supamiu — CHANGES_REQUESTED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13963/ui-diff)

**Review by Supamiu — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2617/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13974/ui-diff)

**Review by GuillaumeNury — APPROVED** [review]

**Pull request merged**